.github/workflows/tests.yml

```diff
@@ -40,6 +42,7 @@ jobs:
           CXX: "sccache clang++"
 
   forest-sync-check:
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
I'm not sure of the usefulness of this. Might as well be `if: false`. But let's merge this and then tinker with the CI later.